src/app/redux/history/reducer.js

```diff
@@ -0,0 +1,51 @@
+export const initialState = {
+  history: [],
+};
+
+export function historyReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'GET_HISTORIES':
+      return state.history;
```

Comment:
Careful here, if you happen to have another value in the store, you store now will be an array and not an object with those keys.

A reducer must always return the whole store which those needed keys modified.